src/py-opentimelineio/opentimelineio/core/__init__.py

```diff
@@ -38,6 +38,32 @@
     item,
 )
 
+__all__ = [
```

Comment:
`__all__` is necessary to document the API as it should be. OTIO uses `from ... import ...` extensively to model the API. So to be representative of that in the reference documentation, we need `__all__`, else Sphinx wouldn't be able to document the class/function imports. As an example, see https://jcmorin-opentimelineio.readthedocs.io/en/latest/api/python/opentimelineio.core.html where all "core" classes are documented.